### PR TITLE
chore: update docs for crs-toolchain

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

File: content/development/crs_toolchain.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: crs-toolchain
+weight: 10
+disableToc: false
+chapter: false
+---
+
+> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go) (which it itself based on the brain melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
```
Comment (fzipi) [inline content/development/crs_toolchain.md:8]:
```suggestion
> The crs-toolchain is the utility belt of CRS developers. It provides a single point of entry and a consistent interface for a range of different tools. Its core functionality (owed to the great [rassemble-go](https://github.com/itchyny/rassemble-go) (which is itself based on the brain-melting [Regexp::Assemble](https://github.com/ronsavage/Regexp-Assemble) Perl module) is to assemble individual parts of a regular expression into a single expression (with some optimizations).
```

File: content/development/regex_assembly.md [hunk 2]
```diff
@@ -1,84 +1,15 @@
...
-## Adjusting the Logging Level
-
-The level of logging can be adjusted with the `--loglevel` option. Accepted values are `DEBUG`, `INFO`, `WARNING`, `ERROR`, and `CRITICAL`.
-
-Not many things are currently written to the log, so it may seem that changing the logging level doesn't have any effect. This is especially true since the processing modules depend on the rule that the script processes and the command that is being run (e.g., `generate` usually produces no log output since stdout is used to print the resulting regular expression). The `cmdline` processor, for example, includes debug log output, but the `assemble` processor currently doesn't.
-
-## Data File Format
-
-The data files (`.data` suffix, under `util/regexp-assemble/data`) contain one regular expression per line. These files are meant to be processed by [regexp-assemble.py](regexp-assemble.py).
+The files containing regular expression specifications (`.data` suffix, under `data`) contain one regular expression per line. These files are meant to be processed by the [crs-toolchain]({{< ref "crs_toolchain" >}}).
```
Comment (fzipi) [inline content/development/regex_assembly.md:12]:
```suggestion
The files containing regular expression specifications (`.ra` suffix, under `data`) contain one regular expression per line. These files are meant to be processed by the [crs-toolchain]({{< ref "crs_toolchain" >}}).
```

**Review by fzipi — CHANGES_REQUESTED** [review]
Maybe also adding that included files can include additional files, and there can be definitions in those files also?

File: content/development/regex_assembly.md (hunk 2 above)
Comment (theseion) [reply content/development/regex_assembly.md:12]: @RedXanadu suggested renaming the `data` directory to something better. Maybe `regex-assembly`? We could make the change now, which will probably lead to less disruption later... What do you think?

Comment (lifeforms) [reply content/development/regex_assembly.md:12]: Or something like `regex-src`? Anyway, it's a bit of bikeshedding. But I think `regexp-assembly` is the most clear (`data` could mean everything and it's also used for pmFromFile) and I also think the `.ra` extension is a GREAT idea.

Comment (theseion) [reply content/development/regex_assembly.md:12]: Thanks @lifeforms, I agree.

**Review by fzipi — APPROVED** [review]

**Pull request merged**